### PR TITLE
Adds high-quality prosthetic limbs to Medbay

### PR DESCRIPTION
**Pull request by TheVekter**
## About The Pull Request
This adds a method for Medical to be able to print improved prosthetic limbs, allowing them to replace limbs without having to make synthflesh. This still has all the negatives of cyborg limbs, so it's not quite as good as growing real limbs. Just gives another option. Eventually, this is going to replace the use of cyborg limbs entirely, but I'll do that in a future PR.

## Why It's Good For The Game
Prosthetic limbs without having to bother a possibly non-existent roboticist. This is more in Medical's wheelhouse anyway.

## Changelog
:cl:
add: Added prosthetic limbs that can be researched and printed in Medbay.
/:cl:

### CODE REVIEW

**Comment by Michael-Ashfield** [comment]
Medbay already has prosthetics which work differently to cyborg limbs, they come in a freezer located at the top of medbay storage next to the sec office. The only thing is I don't believe they can be made after they are used up.

**Comment by Fikou** [comment]
these should be a cargo crate

**Comment by Dorsisdwarf** [comment]
Yeah you probbably should have added the existing prosthetics instead of a brand new type which are better in every way

**Comment by TheVekter** [comment]
> Medbay already has prosthetics which work somewhat similarly to cyborg limbs (Lower stats), they come in a freezer located at the top of medbay storage next to the sec office. The only thing is I don't believe they can be made after they are used up, they also don't appear to be a cargo purchase.
> In addition, the existing prosthetics is what the prosthetic limb quirk uses.
> 
> code/modules/surgery/bodyparts/robot_bodyparts.dm
> 
> /obj/item/bodypart/l_arm/robot/surplus
> /obj/item/bodypart/r_arm/robot/surplus
> /obj/item/bodypart/l_leg/robot/surplus
> /obj/item/bodypart/r_leg/robot/surplus

Yup, I know about those. I added these so they could be made. It basically just replaces the cyborg limb method without me making cyborg limbs printable from the Protolathe.

> these should be a cargo crate

Yes they should! Noted.

> Yeah you probbably should have added the existing prosthetics instead of a brand new type which are better in every way

I've always assumed that the ones you start with if you have the quirk are supposed to be kinda cheap, beat-up ones that (from a gameplay standpoint) are identical to normal limbs. These are easy to make, (slightly) better than cyborg limbs, and are a viable alternative to using the limb grower in situations where synthflesh isn't readily available. If you think the higher brute resistance will make a big enough difference I can always just change that.

**Comment by Michael-Ashfield** [comment]
Could I suggest that you consider making these a type of /obj/item/bodypart/l_arm/robot/surplus, maybe /obj/item/bodypart/l_arm/robot/surplus/upgraded to reduce code repeating and just changing the stats where needed.

Just opinion here: I thought the point of prosthetics being worse than robotic limbs is for inter-departmental co-operation, you have to work together for the good stuff otherwise you can be patched up with the bad stuff.

**Comment by ATH1909** [comment]
> Eventually, this is going to replace the use of cyborg limbs entirely, but I'll do that in a future PR.

Booooooo. Why would you want to nerf full-augging?

**Comment by ExcessiveUseOfCobblestone** [comment]
I'm not sure why we need a cyborg part equivalent in medbay. Just go to robotics?

I wouldn't mind if these found themselves in their own spot (perhaps somewhere between organic and cyborg limbs) instead of just a duplication of cyborg limbs but it's medbay.

**Comment by ATH1909** [comment]
Also, what's the point of the limb grower if these are a thing?

File: code/modules/surgery/bodyparts/bodyparts.dm [hunk 1]
```diff
@@ -732,3 +732,39 @@
 	dismemberable = 0
 	max_damage = 5000
 	animal_origin = DEVIL_BODYPART
+
+obj/item/bodypart/l_arm/prosthetic
+	name = "high-grade prosthetic left arm"
+	desc = "A well-made robotic limb coated in a flesh-like rubber. Able to withstand more damage than a normal human limb."
```
Comment (ATH1909) [inline code/modules/surgery/bodyparts/bodyparts.dm]: *more **brute** damage

**Pull request closed without merging**